internal/handlers/msg_service_router_test.go

```diff
@@ -177,12 +177,12 @@ func TestMsgService(t *testing.T) {
 	assert.Equal(t, "acc_seq", string(res.Events[5].Attributes[0].Key))
 	assert.Equal(t, "tx", res.Events[6].Type)
 	assert.Equal(t, "signature", string(res.Events[6].Attributes[0].Key))
-	assert.Equal(t, "tx", res.Events[7].Type)
-	assert.Equal(t, antewrapper.AttributeKeyAdditionalFee, string(res.Events[7].Attributes[0].Key))
-	assert.Equal(t, "10nhash", string(res.Events[7].Attributes[0].Value))
-	assert.Equal(t, "tx", res.Events[8].Type)
-	assert.Equal(t, antewrapper.AttributeKeyBaseFee, string(res.Events[8].Attributes[0].Key))
-	assert.Equal(t, "190500000nhash", string(res.Events[8].Attributes[0].Value))
```

Comment:
for reference: we decided to change the order of emission of the events for msg fees, since they are charged at the end we thought it better to emit the events at the end of the DeliverTx cycle.